Podfile

```diff
@@ -3,7 +3,7 @@ source 'https://github.com/CocoaPods/Specs'
 workspace 'VideoQuickStart'
 
 abstract_target 'TwilioVideo' do
-  pod 'TwilioVideo', '2.6.0-preview1'
+  pod 'TwilioVideo', '2.6.0-preview2'
```

Comment:
👍 Can you also update the docs links in the master README.md?

Reply:
Lemme go have a look see.

Reply:
Done.